web/js/pages/Home.jsx

```diff
@@ -17,26 +22,42 @@ type Props = {
 }
 
 class Home extends React.Component < Props > {
+  componentWillMount() {
+    document.body.style.margin = 0;
+    document.body.style.backgroundColor = '#F5F5F5';
+  }
+
   render() {
     if (!this.props.viewer.userbank || this.props.viewer.userbank.edges.length === 0) {
       return null;
     }
-    return (<SnackbarErrorContext>
-      <Grid container spacing={16}>
-        <Grid item xs={12} sm={6}>
-          {this.props.viewer.userbank.edges[0]
-            ? <PersonalStatusRelay bank={this.props.viewer.userbank.edges[0].node} />
-            : null}
-        </Grid>
-        <Grid item xs={12} sm={6}>
-          {this.props.viewer.userbank.edges[0]
-            ? <BankAccountRelay bank={this.props.viewer.userbank.edges[0].node} />
-            : null}
-        </Grid>
-        <Grid item xs={12} sm={6}>
-          <InvestBucketGridRelay profile={this.props.viewer.profile} />
-        </Grid>
-      </Grid></SnackbarErrorContext>
+    return (
+      <MuiThemeProvider theme={theme}>
+        <div>
+          <AppBar />
```

Comment:
Can you have the MuiTheme and AppBar to a separate file? Should do something like this:
```
function layout(props) {
  return (
  <MuiThemeProvider>
    <AppBar />
    {props.children}
  </MuiThemeProvider>
  );
}
```

Then in relay/routes.jsx wrap the existing route in this:
```
<Route
  path="/"
  Component={Layout}
>
  {<<<Previous stuff>>>}
</Route>
```